### ISSUE TITLE
Unpack error in reducer

### ISSUE THREAD

**Issue by itssuyog96** [ankschoubey/Hadoop-Mini-Project#3]
Getting this with python 2.7 as well as python 3

```
Traceback (most recent call last):
  File "/Users/suyog/Projects/Hadoop-Mini-Project/./value_summation_reducer.py", line 14, in <module>
    word, count = line.split('\t')
ValueError: need more than 1 value to unpack
```

**Comment by itssuyog96** [comment]
Temporarily fixed to avoid the interrupt between execution.

```
 # parse the input we got from mapper.py
    try:
        word, count = line.split('\t')
    except ValueError:
        #unable to split output
        continue
```

Tried to print the contents of `line`, but I think the output stream of hadoop stream is blocking the output stream of mapper.py. Any idea how to know the contents of `line`.

**Comment by itssuyog96** [comment]
Finally did this,

```
line = line.strip('(')
line = line.strip(')')
line = line.strip("\'")
word, count = line.split("\', \'")[0], line.split("\', \'")[-1]
```

Now it works, but not sure if it will work on other hadoop versions.

**Comment by ankschoubey** [comment]
I guess  ```#!/usr/bin/python``` this line which is at the beginning of mapper and reducer is running it with python 2 so

the output which is

```(1\t2)```
instead of

```1\t2```

That might be one of the issue.

Just print and paste the content of line before ```word, count = line.split("\', \'")[0], line.split("\', \'")[-1]```, I'll fix it.

**Comment by ankschoubey** [comment]
Found it..

the output of mapper was 

```
('N', '\t', '36067')
('Y', '\t', '242674')
('Y', '\t', '193066')
('Y', '\t', '220314')
````

instead of 

```
N 	 36067
Y 	 242674
Y 	 193066
Y 	 220314
```

There are 2 ways to fix this ...
1. Changing ```#!/usr/bin/python``` to python 3. terminal command: ```which python``` 
2. Removing brackets from print statement in mapper and reducer so that they both will run in python 2. 
3. Also try just removing ```#!/usr/bin/python``` so that it will use default python. But removing that line did cause me bugs when I tested things on docker. even though python was mapped to same location ```/usr/bin/python```
